src/Listener/FixtureEvent.php

```diff
@@ -22,9 +22,10 @@ final class FixtureEvent
 
     private FixtureInterface $fixture;
 
-    /** @var array */
-    private $fixtureOptions;
+    /** @var array<mixed> */
+    private array $fixtureOptions;
 
+    /** @param array<mixed> $fixtureOptions */
     public function __construct(SuiteInterface $suite, FixtureInterface $fixture, array $fixtureOptions)
```

Comment:
Probably outside of scope of current PR, but this should become part of ECS config.

```suggestion
    /** @param array<mixed> $fixtureOptions */
    public function __construct(private SuiteInterface $suite, private FixtureInterface $fixture, private array $fixtureOptions)
```

and 
```patch
---    private SuiteInterface $suite;
---
---    private FixtureInterface $fixture;
---
---    /** @var array<mixed> */
---    private array $fixtureOptions;
```